test/run_test.py

```diff
@@ -185,6 +187,181 @@ def __contains__(self, item):
     "distributed/_tensor/test_attention",
 ]
 
+S390X_BLOCKLIST = [
```

Comment:
From the OH: The blocklist here it too long, maybe we could provide an allow list here instead, i.e. like running only test X, Y, Z